TeachingRecordSystem/tests/TeachingRecordSystem.SupportUi.Tests/PageTests/Persons/PersonDetail/ChangeLogTests.cs

```diff
@@ -42,10 +45,10 @@ public async Task Get_WithPersonIdForPersonWithNoNotes_DisplaysNoChanges()
     }
 
     [Fact]
-    public async Task Get_WithPersonIdForPersonWithNotes_DisplaysChangesInDescendingOrder()
+    public async Task Get_WithPersonIdForPersonWithChanges_DisplaysChangesInDescendingOrder()
```

Comment:
I think we should split this test up and have one for each type of item/event.

Reply:
Done